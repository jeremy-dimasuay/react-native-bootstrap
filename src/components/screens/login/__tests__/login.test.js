import React from 'react';
import renderer from 'react-test-renderer';

import { Login } from '../';

function createLogin(props = {}) {
    return (
        <Login
            navigation={{}}
            {...props}
        />
    );
}
describe('<Login>', () => {
    it('renders', () => {
        const component = renderer.create(createLogin());
        expect(component.toJSON()).toMatchSnapshot();
    });
});
