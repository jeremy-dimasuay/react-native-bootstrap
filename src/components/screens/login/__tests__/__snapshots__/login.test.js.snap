// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login> renders 1`] = `
<View
  style={
    Array [
      Object {},
      undefined,
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Array [
        Object {},
        undefined,
      ]
    }
  >
    Login please
  </Text>
  <View
    accessibilityComponentType="button"
    accessibilityLabel={undefined}
    accessibilityTraits={
      Array [
        "button",
      ]
    }
    accessible={true}
    collapsable={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <View
      style={
        Array [
          Object {},
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={undefined}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#007AFF",
              "fontSize": 18,
              "padding": 8,
              "textAlign": "center",
            },
          ]
        }
      >
        Login
      </Text>
    </View>
  </View>
</View>
`;
